feat: added docker-compose for running postgres (#39)
EddieHubCommunity/CreatorsRegistry: README.md

5. Seed the database by inserting example data with `npm run db:dev:seed` (if you need to reset the database, use `npm run db:dev:reset`).
6. Start the local development server with `npm run dev`.

### Postgres using Docker Compose

Note: you must have docker installed 

> Run the docker command `docker-compose up -d`

Reviewer: ```suggestion
Run the docker command `docker compose up -d`
```

5. Seed the database by inserting example data with `npm run db:dev:seed` (if you need to reset the database, use `npm run db:dev:reset`).
6. Start the local development server with `npm run dev`.

> ### Run Postgres using docker 
> 
> #### Requirement 
> - Must have docker installed 
> ```bash
> docker-compose up -d
> ``` 
> 

Reviewer 2: ```suggestion
### Postgres using Docker Compose

Note: you must have docker installed 

Run the docker command `docker-compose up -d`

```
